src/utils/api/constants.ts

```diff
@@ -1,2 +1,4 @@
 // TODO: Make this configurable based on prod/dev run status
 export const BASEURL = "http://localhost:3080/nxt?"; // proxy runs on 3080 by default
```

Comment:
consider renaming this to something more specific to the NXT/JUP api endpoint. as it stands right now this was put into constants so that other apis could be utilized. but the name of this variable is very vague and generic which might lead to some confusion in the future. anytime tho it's easy to refactor the name of this, so it's very low priority.